app/views/resources/_upload_localized_resource_template.haml

```diff
@@ -28,6 +28,12 @@
           %input{:type => 'file', 'data-bind' => 'fileupload: url, fileuploadAdd: add, fileuploadprogressall: showProgress'}
         Replace
       / /ko
+      / ko if: hasAudio() && (uploadStatus() == 'ok' || uploadStatus() == 'standBy')
+      %br
+      %audio{'controls' => true, "style" => "width: 450px", "preload" => "none"}
```

Comment:
Do we care about compatibility with old browsers? What does this do if the browser doesn't support the `<audio>` element or the audio format?